# HG changeset patch
# User Philipp Lohmann [pl] <Philipp.Lohmann@Sun.COM>
# Date 1272292577 -7200
# Node ID 1f21040f063df81e894459971d03424146203339
# Parent  3f6d3fa8969b69a4ecdb2926d962d2e4f8056678
vcl111: #i103999# fix problem with moblin theme (thanks tml)

diff -r 3f6d3fa8969b -r 1f21040f063d vcl/unx/gtk/gdi/salnativewidgets-gtk.cxx
--- vcl/unx/gtk/gdi/salnativewidgets-gtk.cxx	Mon Apr 26 11:14:03 2010 +0200
+++ vcl/unx/gtk/gdi/salnativewidgets-gtk.cxx	Mon Apr 26 16:36:17 2010 +0200
@@ -3475,7 +3475,7 @@
     aStyleSet.SetMenuColor( aBackColor );
     aStyleSet.SetMenuTextColor( aTextColor );
 
-    aTextColor = getColor( pMenubarStyle->text[GTK_STATE_NORMAL] );
+    aTextColor = getColor( pMenubarStyle->fg[GTK_STATE_NORMAL] );
     aStyleSet.SetMenuBarTextColor( aTextColor );
     
 #if OSL_DEBUG_LEVEL > 1
